lib/bgs/form686c.rb

```diff
@@ -96,8 +97,26 @@ def get_state_type(payload)
       'Started'
     end
 
+    # if we are setting the claim to be manually reviewed
+    # and the Veteran is currently receiving pension benefits
+    # set the claim type code to 130DAEBNPMCR (PMC eBenefits Dependency Adjustment Reject)
+    # else use 130DPNEBNADJ (eBenefits Dependency Adjustment)
+    def get_claim_type(proc_state)
+      if proc_state == 'MANUAL_VAGOV'
+        pension_response = bid_service.get_awards_pension
+        receiving_pension = pension_response.body['awards_pension']['is_in_receipt_of_pension']
+        return '130DAEBNPMCR' if receiving_pension
+      end
+
+      '130DPNEBNADJ'
```

Comment:
Since we are doing an explicit return on L108, should we do one here also?

Reply:
> Since we are doing an explicit return on L108, should we do one here also?

Rubocop does not like us to use `return` if its not needed.

Reply:
👌